scripts/quests/ahtUrhgan/The_Prince_and_the_Hopper.lua

```diff
@@ -123,19 +132,32 @@ quest.sections =
                 end,
             },
 
+            onZoneIn = {
```

Comment:
Do you have an updated video of this quest?  Watched a previous capture, and there is no onZoneIn element for this cutscene.

Reply:
very sorry for the long delay - ive been mostly without my main pc that has lsb on it.

i don't have video, just the capture data (will also have the levels here ive changed) https://www.dropbox.com/s/w2csr24dttvvbk3/prince%20and%20hopper%20data.rar?dl=0
but this would be irrelevant if the setpos is unnecessary for kicking off the next cs and just returns on finish.
im unsure about cs 20 which is another cs kicked off by zone change and was missing here entirely. i will need to validate/install lsb to test. 
i was doing cs 222 - setpos zone in to wajoam -- kicks of cs 20 -- setpos zone back to mamook -- kicks off 227
the current lsb file has cs 222 - kicks off 227

Reply:
We've had a bit of event changes since the initial PR was posted, but you'll definitely need to change zones if there's a queue.  I'll take a look at some things to see if there's some other ideas to toss out.